dpr/indexer/faiss_indexers.py

```diff
@@ -111,40 +124,45 @@ def __init__(self, vector_sz: int, buffer_size: int = 50000, store_n: int = 512
         index.hnsw.efSearch = ef_search
         index.hnsw.efConstruction = ef_construction
         self.index = index
-        self.phi = 0
+        self.phi = None
 
-    def index_data(self, data: List[Tuple[object, np.array]]):
-        n = len(data)
+    def index_data(self, vector_files: List[str]):
+        self._set_phi(vector_files)
```

Comment:
you are reading input files twice in this index_data method.
Which is probably fine since we don't keep all data from files in ram.